examples/step-22/step-22.cc

```diff
@@ -694,12 +692,15 @@ namespace Step22
                 for (unsigned int j = 0; j <= i; ++j)
                   {
                     local_matrix(i, j) +=
-                      (2 * (symgrad_phi_u[i] * symgrad_phi_u[j]) -
-                       div_phi_u[i] * phi_p[j] - phi_p[i] * div_phi_u[j]) *
-                      fe_values.JxW(q);
+                      (2 * (symgrad_phi_u[i]      // ((2 * grad^s v_i(x_q)
+                            * symgrad_phi_u[j])   //     * grad^s u_j(x_q))
+                       - div_phi_u[i] * phi_p[j]  // - div v_i(x_q) * p_j(x_q)
+                       - phi_p[i] * div_phi_u[j]) // - q_i(x_q) * div u_j(x_q))
+                      * fe_values.JxW(q);         // * dx
```

Comment:
nice job with the formatting.

But the symbol names aren't quite correct. It's not `v_i,u_j` but really `phi_i`, `phi_j`. Similarly for `p_j` and `q_i` -- we should be referencing the name of the shape functions we use in the introduction -- they're not different `p`s and `q`s, but the same.

Reply:
Then, let me just remove the comments altogether. Otherwise, I am just repeating what is already written in all the lines and this makes keeping the 80 character limit just much harder.

Reply:
Can you put the (longer) comments into separate lines that precede the parts of the formula? Would that be a way to keep things that belong together together?

Reply:
Do you like
```
      local_matrix(i, j) +=
                       // ((2 * grad^s phi_u_i(x_q) * grad^s phi_u_j(x_q))
                       (2 * (symgrad_phi_u[i] * symgrad_phi_u[j])
                        // - div phi_u_i(x_q) * phi_p_j(x_q)
                        - div_phi_u[i] * phi_p[j]
                        // - phi_p_i(x_q) * div phi_u_j(x_q))
                        - phi_p[i] * div_phi_u[j]) //
                       * fe_values.JxW(q);         // * dx
```
better? Or did I misunderstand what you are proposing instead?

Reply:
Yes, something like this. I think for symmetry, I'd move the opening and closing parentheses of the bilinear form onto lines of their own, though.